app/js/components/identifiers/addIdentifier.jsx

```diff
@@ -145,12 +146,11 @@ class NewIdentifier extends React.Component {
         {!this.state.isNewIdentifier && !this.props.stateData.editState &&
           <div className="form-group">
             <div className="col-sm-offset-2 col-sm-2 btn-margin">
-              <button
-                type="addButton"
-                name="add"
```

Comment:
Was the name attribute not being used?

Reply:
Its usage is just to help in identification ....